app/models/amt/AMTAssignmentTable.scala

```diff
@@ -32,5 +34,8 @@ object AMTAssignmentTable {
       (amtAssignments returning amtAssignments.map(_.amtAssignmentId)) += asg
     asgId
   }
+  def getConfirmationCode(workerId: String): String = db.withTransaction { implicit session =>
```

Comment:
This piece of code was going to be altered. But just to be clear, that one HIT will have as many missions as the user can do before being logged out (either because they were working in incognito mode and closed the browser, if they manually logged out, ...). If we do have users working on multiple HITs it'll be on 2 different kinds of HITs. The first kind is what we have right now (with a base reward on completing the first mission). We assign a qualification to users who complete this HIT which will allow them to work on the second kind of HITs. These will have a base reward of 0 and we specify that this is a way to log back into Project sidewalk with their turker account and that they will continue to be paid with bonuses as before.

Reply:
I'm not entirely clear on how this addresses my concern. I now understand how turkers will be able to continue auditing in another session, cool! But.. what about this code right here..??

Reply:
I've changed it. The function will now take assignmentId as an input along with the workerId